src/plugins/saved_objects_management/server/routes/scroll_count.ts

```diff
@@ -42,6 +42,7 @@ export const registerScrollForCountRoute = (router: IRouter) => {
           namespacesToInclude: schema.maybe(schema.arrayOf(schema.string())),
           searchString: schema.maybe(schema.string()),
           workspaces: schema.maybe(schema.arrayOf(schema.string())),
+          availiableWorkspaces: schema.maybe(schema.arrayOf(schema.string())),
```

Comment:
typo: available

Reply:
thanks for catching this, this has been fixed.